### PR TITLE
hotfix(panels): 12585 - fix min-height values and add matrix to desktop layout

### PR DESCRIPTION
**Pull request by IlyaIzr**

### CODE REVIEW

**Review by propakov — APPROVED** [review]
👍

File: src/features/analytics_panel/components/AnalyticsPanel/AnalyticsPanel.tsx [hunk 1]
```diff
@@ -19,7 +19,7 @@ const LazyLoadedAnalyticsPanelHeader = lazy(
 export function AnalyticsPanel() {
   const [isOpen, setIsOpen] = useState<boolean>(true);
   const isMobile = useMediaQuery(IS_MOBILE_QUERY);
-  const minHeight = 100;
+  const minHeight = 119;
```
Comment (Akiyamka) [inline src/features/analytics_panel/components/AnalyticsPanel/AnalyticsPanel.tsx]: Looks like feature constant

File: src/features/events_list/components/EventsListPanel/EventsListPanel.tsx [hunk 2]
```diff
@@ -44,7 +44,7 @@ export function EventsListPanel({
   const isLaptop = useMediaQuery(IS_LAPTOP_QUERY);
   const virtuoso = useRef(null);
   const [{ data: userModel }] = useAtom(userResourceAtom);
-  const minHeight = 190;
+  const minHeight = 279;
```
Comment (Akiyamka) [inline src/features/events_list/components/EventsListPanel/EventsListPanel.tsx]: Looks like feature constant

File: src/features/layers_panel/components/MapLayersPanel/MapLayersPanel.tsx [hunk 3]
```diff
@@ -17,7 +17,7 @@ export function MapLayerPanel() {
   const [isOpen, setIsOpen] = useState<boolean>(true);
   const turnOffTooltip = useAction(currentTooltipAtom.turnOffById);
   const isMobile = useMediaQuery(IS_MOBILE_QUERY);
-  const minHeight = 40;
+  const minHeight = 79;
```
Comment (Akiyamka) [inline src/features/layers_panel/components/MapLayersPanel/MapLayersPanel.tsx]: Looks like feature constant

File: src/features/legend_panel/components/LegendPanel/LegendPanel.tsx [hunk 4]
```diff
@@ -22,7 +22,7 @@ export function LegendPanel({ layers }: LegendPanelProps) {
   const [isOpen, setIsOpen] = useState<boolean>(false);
   const isMobile = useMediaQuery(IS_MOBILE_QUERY);
   const turnOffTooltip = useAction(currentTooltipAtom.turnOffById);
-  const minHeight = 100;
+  const minHeight = 79;
```
Comment (Akiyamka) [inline src/features/legend_panel/components/LegendPanel/LegendPanel.tsx]: Looks like feature constant

File: src/features/legend_panel/components/LegendPanel/LegendPanel.module.css [hunk 5]
```diff
@@ -61,6 +61,18 @@
   margin-left: auto;
 }
 
+/* Desktop only */
+@media screen and (min-width: 1900px) {
+  /* Prevent resizing out of parent container */
+  .panelContainer.isOpen {
+    min-height: 120px;
```
Comment (Akiyamka) [inline src/features/legend_panel/components/LegendPanel/LegendPanel.module.css]: Why we have min height in jsx for other panels but not here?

**Pull request merged**